src/DotNet/Importer.cs

```diff
@@ -827,11 +833,11 @@ public Importer(ModuleDef module, ImporterOptions options, GenericParamContext g
 			case ElementType.ByRef:		result = new ByRefSig(Import(type.Next)); break;
 			case ElementType.ValueType: result = CreateClassOrValueType((type as ClassOrValueTypeSig).TypeDefOrRef, true); break;
 			case ElementType.Class:		result = CreateClassOrValueType((type as ClassOrValueTypeSig).TypeDefOrRef, false); break;
-			case ElementType.Var:		result = new GenericVar((type as GenericVar).Number, gpContext.Type); break;
+			case ElementType.Var:		result = new GenericVar((type as GenericVar).Number, gpContext.Type ?? (type as GenericVar).OwnerType); break;
```

Comment:
`type as GenericVar` could be `null`. Same below (see MVar below).

Reply:
But dnlib already uses `(type as GenericVar).Number`, if it's null, NRE already be thrown before my code get executed.